drivers/copy/copy_linux.go

```diff
@@ -297,5 +297,8 @@ func doCopyXattrs(srcPath, dstPath string) error {
 	// this function is used to copy those. It is set by overlay if a directory
 	// is removed and then re-created and should not inherit anything from the
 	// same dir in the lower dir.
-	return copyXattr(srcPath, dstPath, "trusted.overlay.opaque")
+	if err := copyXattr(srcPath, dstPath, "trusted.overlay.opaque"); err != nil {
```

Comment:
This seems like you are still doing both.  I don't see a check for rootless versus rootfull?  If I am running rootless, will the first copyXattr fail and return an error and never get to the second?

Reply:
the first copy wouldn't be performed as the xattr is missing.

I thought there could be cases where the copy runs as root and misses to copy the `user.*` xattr if present.

I can simplify it and check `unshare.IsRootless()`